backend/src/packages/projects/project.service.ts

```diff
@@ -8,7 +8,9 @@ import {
   type ProjectGetAllResponseDto,
 } from './libs/types/types.js';
 
-class ProjectService implements Omit<IService, 'find' | 'update' | 'delete'> {
+class ProjectService
+  implements Omit<IService, 'find' | 'create' | 'update' | 'delete'>
```

Comment:
```suggestion
  implements Omit<IService, 'find' | 'update' | 'delete'>
```

Reply:
@what1s1ove but if I do this I will have a problem with create method, because Interface has one attribute - payload. But I have two attrs - userId and ProjectCreateRequestDto. That is why I created the method createByUserId before like findByUserId in this service

Reply:
You can change the type of payload. On the frontend 1st one, on the backend 2d one

Reply:
@what1s1ove  But I removed userId from ProjectCreateRequestDto payload, because you wrote to remove it and add userId on the backend. So I take the user from the request and pass the userId attribute from the controller to the service method

Reply:
You don't need to use the same types on the backend end frontend
Even more. you don't need to use the same type for controller and service. You can extends them if you need it